spec/components/cookie_banner_spec.rb

```diff
@@ -51,15 +57,24 @@ def component_name
       confirmation_message: "You’ve accepted all cookies.",
     )
     assert_select ".gem-c-cookie-banner__message .govuk-heading-m", text: "Can we store analytics cookies on your device?"
-    assert_select ".gem-c-cookie-banner__message .govuk-body", text: "This is some custom text with a link to the cookies page"
+    assert_select ".govuk-cookie-banner__content", text: "This is some custom text with a link to the cookies page"
     assert_select ".govuk-link[href='/cookies']", text: "cookies page"
     assert_select ".gem-c-cookie-banner__confirmation-message", text: "You’ve accepted all cookies."
   end
 
+  it "renders with multi paragraph content" do
+    render_component(
+      title: "Can we store analytics cookies on your device?",
+      text: ["Paragraph 1", "Paragraph 2"],
+      confirmation_message: "You’ve accepted all cookies.",
+    )
+    assert_select "p.govuk-body", text: "Paragraph 1"
+    assert_select "p.govuk-body", text: "Paragraph 1"
```

Comment:
Looks like a copy/paste oversight here - this line should say `text: Paragraph 2`

Reply:
Yup. Fixed.